ajax/libs/mui/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "mui",
+  "filename": "js/mui.min.js",
+  "description": "A effective front-end framework closest to the native app",
```

Comment:
I think there are problems here.

Should `closest` be used with `the`? The `native` here actually means its performance which is really close to the native application, but I can't see the correct meaning in the description here.